README.md

```diff
@@ -69,6 +69,7 @@ ext.gitVersioner = [
         yearFactor              : 1200, 	  // default "1000", increasing every 8.57h
         snapshotEnabled         : false,      // default false, the "-SNAPSHOT" postfix
         localChangesCountEnabled: false,       // default false, the (<commitCount>) before -SNAPSHOT
+		initialOffset           : 2100000,    // default "0", amount of time (in seconds) to offset the initial commit by, if development started before commits began
```

Comment:
Wouldn't it be better to set an exact timestamp instead of a difference compared to the first commit?